app/shared/components/Producers.js

```diff
@@ -234,7 +234,9 @@ class Producers extends Component<Props> {
       );
     }
 
-    if (!isValidUser && settings.walletMode !== 'wait') {
+    if (!isValidUser
+
+      && settings.walletMode !== 'wait') {
```

Comment:
Did something change here? What's up with the formatting?

Reply:
😅 I was playing with that conditional, must have forgotten to completely revert the change, but it's all good now